cpp/src/toppra/geometric_path.hpp

```diff
@@ -52,7 +52,7 @@ class PiecewisePolyPath : public GeometricPath {
    * @param coefficients Polynoamial coefficients.
    * @param breakpoints Vector of breakpoints.
    */
-  PiecewisePolyPath(const Matrices &, const std::vector<value_type> &);
+  PiecewisePolyPath(const Matrices &, std::vector<value_type> );
```

Comment:
Why don't you want to pass a const reference ?

Reply:
Hm. This looks quite unfamiliar. I should have checked the linter fixes more carefully.

Reply:
I wonder why it changes const reference to pass-by-value though.

Reply:
I came to the same conclusion. But if they are the same, I don't understand why it changed it (and, even more, why did it decide to use a C++11 only feature when a C++98 do the same job)... But there might be a good reason. Anyway, I let you keep the one you like.